sonarqubeandjacoco/pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
The entire file should use spaces for indentation instead of tabs. There should be a standard formatter in the tutorials repo that you can import into your IDE. There's one for eclipse and one for intellij.